integration_testing/features/coach/coach-class-reports-quizzes.feature

```diff
@@ -68,8 +68,10 @@ Feature: Coach reviews learner reports for quizzes
   Scenario: A learner has completed a quiz
     When a learner has completed the <quiz>
     Then their *Progress* column says *Completed*
+      And their *Progress* column says *N questions answered* or *All questions answered*
```

Comment:
Can't comment on line 65 above, but it should read `Then the learner's *Progress* column displays *X of Y questions answered*

Reply:
I updated this - it won't show as resolved on the PR because the comment is on a different line - but please check it out and if there is an issue please let me know @radinamatic